tests/ref_outputs/test016/params_system.cfg

```diff
@@ -10,7 +10,7 @@ segments = [
 	(-0.5, 0.1),
 	]
 tramp = 1e-3
-prevDir = /home/cogswell/Projects/mpet/tests/test_outputs/20200430_092037/test016/../test008/sim_output
+prevDir = tests/ref_outputs/test008/sim_output
```

Comment:
Should it always point to the ref_outputs, even when running somewhere else?
Otherwise maybe 
```
prevDir = ../test008/sim_output 
```
is better

Reply:
That's a good point. mpet treats a directory here relative to PYTHONPATH, but it probably should be relative to the config file.

Reply:
I think so too, should I make a separate issue for this?